var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -40,10 +40,12 @@ class Legion(CMakePackage):
     variant('mpi', default=True,
             description='Build on top of mpi conduit for mpi inoperability')
     variant('shared', default=True, description='Build shared libraries')
+    variant('hdf5', default=True, description='Enable HDF5 support')
 
     depends_on("cmake@3.1:", type='build')
     depends_on("gasnet~aligned-segments~pshm segment-mmap-max='16GB'", when='~mpi')
     depends_on("gasnet~aligned-segments~pshm segment-mmap-max='16GB' +mpi", when='+mpi')
+    depends_on("hdf5~mpi", when='+hdf5')
```

Comment:
@tuxfan does this only work with `hdf5~mpi` or would `hdf5` also work? If you're goal is to avoid building an `mpi` implementation to save time, you can specify that on the command line like `spack install legion+hdf5 ^hdf5~mpi`.

Reply:
No, legion can just not link the parallel hdf5 library.